src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -123,14 +123,16 @@ class BankAccountStep extends React.Component {
                             <Text style={[styles.mh5, styles.mb5]}>
                                 {this.props.translate('bankAccount.toGetStarted')}
                             </Text>
-                            <MenuItem
-                                icon={Bank}
-                                title={this.props.translate('bankAccount.logIntoYourBank')}
-                                onPress={() => {
-                                    this.setState({bankAccountAddMethod: CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID});
-                                }}
-                                shouldShowRightIcon
-                            />
+                            {!this.props.isPlaidDisabled && (
```

Comment:
Not well-versed in this topic, but if Plaid is disabled is there anything we can do for the customer? Should we disable this menu item and have a tooltip like "Reach out to concierge to re-enable this" ?

Reply:
Great catch as I just forgot about this :0

We have some existing copy for this but it didn't make it into this PR

>Due to a high number of login attempts, this option has been temporarily disabled for 24 hours. Please try again later or manually enter details instead.

So I'll add that now instead of disabling the option completely